src/main/webapp/js/index.js

```diff
@@ -159,34 +159,36 @@ $(document).ready(function() {
         setSliderHandle(handle, this.value);
       });
 
-      input.addEventListener('keydown', function(e) {
+      input.addEventListener('keydown', function(event) {
         const values = keypressSlider.noUiSlider.get();
         const value = Number(values[handle]);
         const steps = keypressSlider.noUiSlider.steps();
         const step = steps[handle];
         let position;
 
-        // 13 is enter, 38 is key up, 40 is key down.
-        switch (e.which) {
-          case 13:
+        const enter = 13;
+        const up = 38;
+        const down = 40;
```

Comment:
From the style guide: Constant names use `CONSTANT_CASE`

https://google.github.io/styleguide/jsguide.html#naming-constant-names

Reply:
So the constants above should be `ENTER`, `UP` and `DOWN`?